+dj/Schema.m

```diff
@@ -7,7 +7,7 @@
     properties(SetAccess = private)
         package    % the package (directory starting with a +) that stores schema classes, must be on path
         dbname     % database (schema) name
-        prefix=''  % optional table prefix, allowing multiple schemas per database
+        prefix=''  % optional table prefix, allowing multiple schemas per database -- remove this feature if not used
```

Comment:
yes, we need to remove it.  Only Florian has used when they had a very unreasonable constraint: they were now allowed to create more databases on their server.